app/helpers/degrees_helper.rb

```diff
@@ -5,7 +5,7 @@ module DegreesHelper
 
   def degree_type_options
     to_enhanced_options(degree_type_data) do |ref_data|
-      synonyms = (ref_data[:synonyms] || []) << ref_data[:abbreviation]
+      synonyms = (([ref_data[:match_synonyms]] + [ref_data[:suggestion_synonyms]] + [ref_data[:abbreviation]]).flatten || [])
```

Comment:
I'm using `flatten` here as I wrapped the synonym arrays in another array. This was to handle `nil` values when trying to add them together. Might be there is a better way?